packages/protocol/test-sol/unit/common/Accounts.t.sol

```diff
@@ -200,7 +193,12 @@ contract AccountsTest is Test, TestConstants {
       );
   }
 }
-
+contract TransitionToL2AfterL1 is AccountsTest {
```

Comment:
Use the new `WhenL2` pattern, see e.g. in [Validators test](https://github.com/celo-org/celo-monorepo/blob/release/core-contracts/12/packages/protocol/test-sol/unit/governance/validators/Validators.t.sol#L434).